tests/baselines/reference/jsdocFunctionTypeFalsePositive.errors.txt

```diff
@@ -1,24 +1,12 @@
+/a.js(1,13): error TS1098: Type parameter list cannot be empty.
 /a.js(1,14): error TS1139: Type parameter declaration expected.
-/a.js(1,17): error TS1003: Identifier expected.
-/a.js(1,17): error TS1110: Type expected.
-/a.js(1,17): error TS8024: JSDoc '@param' tag has name '', but there is no parameter with that name.

```

Comment:
This function is:

```
/** @param {<} x */
function f(x) {}
```

Clearly this has a parameter named `x`, and now we no longer spuriously complain about that.